frontend/public/components/utils/kebab.tsx

```diff
@@ -484,7 +483,7 @@ export const ResourceKebab = connectToModel((props: ResourceKebabProps) => {
       isDisabled={
         isDisabled !== undefined ? isDisabled : _.get(resource.metadata, 'deletionTimestamp')
       }
-      hoverMessage={hoverMessage ? hoverMessage : t('public~Resource is being deleted.')}
+      hoverMessage={hoverMessage}
```

Comment:
@spadgett I think this will be only shown when,
- `isDisabled` is set to a truthy value by the user, or,
- the `deletionTimestamp` is not a false-y value (PTAL https://github.com/openshift/console/pull/11860/files#diff-4c691cf32ae41978379ee0d56c83d114e86c5a409aae6decf72f25af657e478cR484),

hence fulfilling the ask here, IIUC.

Reply:
Done: https://github.com/openshift/console/pull/11860/commits/23b2e310bf4d9f7059d631466cd56bcd75e36290.